gax-grpc/src/test/java/com/google/api/gax/grpc/GrpcCallContextTest.java

```diff
@@ -46,22 +46,24 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import org.junit.Rule;
 import org.junit.Test;
-import org.junit.rules.ExpectedException;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 import org.mockito.Mockito;
 import org.threeten.bp.Duration;
 
 @RunWith(JUnit4.class)
 public class GrpcCallContextTest {
-  @Rule public ExpectedException thrown = ExpectedException.none();
 
   @Test
   public void testNullToSelfWrongType() {
-    thrown.expect(IllegalArgumentException.class);
-    GrpcCallContext.createDefault().nullToSelf(FakeCallContext.createDefault());
+    Exception actualException = null;
+    try {
+      GrpcCallContext.createDefault().nullToSelf(FakeCallContext.createDefault());
+    } catch (Exception ex) {
+      actualException = ex;
+    }
+    Truth.assertThat(actualException).isInstanceOf(IllegalArgumentException.class);
```

Comment:
The pattern we use here is

try {
  doThingThatThrowsException();
  Assert.fail();
} catch (ExceptionType expected) {
  Assert.assertNotNull(expected.getMessage); // or something more interesting
}

Reply:
Thanks for the review, I have updated try/catch block in the same format. PTAL